app/page.tsx

```diff
@@ -1,44 +1,297 @@
+'use client';
+
+import GET from '@/api/get';
+import ArrowRightIcon from '@/components/icons/ArrowRightIcon';
+import GameIcon from '@/components/icons/GameIcon';
+import NewsCard from '@/components/shared/NewsCard';
+import RectangleSkeleton from '@/components/shared/RectangleSkeleton';
+import ThreadsCard from '@/components/shared/ThreadsCard';
+import { parseVideoId } from '@/utils/functions';
+import { MatchDetail, News, Threads } from '@/utils/types';
+import Image from 'next/image';
+import Link from 'next/link';
+import { useEffect, useState } from 'react';
+
 export default function Home() {
+  const [livestreams, setOngoingLivestreams] = useState<MatchDetail[]>();
+  const [news, setNews] = useState<News[]>();
+  const [threads, setThreads] = useState<Threads[]>();
+
+  const ongoingLivestreams = livestreams?.filter(({ status }) => {
```

Comment:
this should be get from the api query aja......... the api was ready enough for the livestream?status=ongoing/completed/etc